### DISCUSSION TITLE
GeoJSON Upload in lokaler Projektion

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#202] [Ideas]
In QGIS wird der Projekt-EPSG-Code auch standardmäßig beim Export von GeoJSON vorausgewählt und mitgespeichert. Unsere QGIS-Community wünscht sich eine Möglichkeit, auch diese GeoJSON in WebGIS zu laden. Im Anhang ist ein Beispiel für 2 Grundstücke angegeben.
Bitte um Abschätzung.

Anhang:
[2gst_2ST.json](https://github.com/user-attachments/files/17187355/2gst_2ST.json)

**Comment by gis-eni** [comment]
Bereits umgesetzt
